src/main/java/io/github/ryotan/code/rs/inject/CodeParamConverter.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * These codes are licensed under CC0.
+ *
+ * http://creativecommons.org/publicdomain/zero/1.0/deed
+ * http://creativecommons.org/publicdomain/zero/1.0/deed.ja
+ */
+
+package io.github.ryotan.code.rs.inject;
+
+import javax.ws.rs.ext.ParamConverter;
+
+import io.github.ryotan.code.Code;
+import io.github.ryotan.code.CodeEnum;
+
+public class CodeParamConverter<C extends CodeEnum<C>> implements ParamConverter<C> {
+
+
+    private final Class<C> code;
+
+    public CodeParamConverter(Class<C> code) {
+        this.code = code;
+    }
+
+    @Override
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method 'fromString' is not designed for extension - needs to be abstract, final or empty.](https://www.codacy.com/app/ryotan/code-enum-support/file/2818755866/issues/source?bid=3375378&fileBranchId=3375797#l24)